packages/app/src/app/pages/NewDashboard/Content/routes/Templates/index.tsx

```diff
@@ -25,7 +25,7 @@ export const Templates = () => {
     return <Text>loading</Text>;
   }
 
-  const templates = data && data.me && data.me.templates;
+  const templates = data.me.templates;
```

Comment:
I guess this was an intentional check as we have seen crash reports. Also, in my opinion it will be good to have a defensive code so that it doesn't break in the future.

Reply:
Like I said in my review (https://github.com/codesandbox/codesandbox-client/pull/3952#discussion_r411987957), we know we don't have an `error` or `loading` at this point.
This means `data` is present.
Since `data` is an object with a `me` object (which has  a `templates` array), we can simplify the code to what I've done